File: decompiled/sources/com/hjq/http/EasyLog.java
package com.hjq.http;

public final class EasyLog {
    public static void print() {
        print("----------------------------------------");
    }

    public static void print(String str) {
        if (EasyConfig.getInstance().isLogEnabled()) {
            EasyConfig.getInstance().getLogStrategy().print(str);
        }
    }

    public static void json(String str) {
        if (EasyConfig.getInstance().isLogEnabled()) {
            EasyConfig.getInstance().getLogStrategy().json(str);
        }
    }

    public static void print(String str, String str2) {
        if (EasyConfig.getInstance().isLogEnabled()) {
            EasyConfig.getInstance().getLogStrategy().print(str, str2);
        }
    }

    public static void print(Throwable th) {
        if (EasyConfig.getInstance().isLogEnabled()) {
            EasyConfig.getInstance().getLogStrategy().print(th);
        }
    }

    public static void print(StackTraceElement[] stackTraceElementArr) {
        if (EasyConfig.getInstance().isLogEnabled()) {
            EasyConfig.getInstance().getLogStrategy().print(stackTraceElementArr);
        }
    }
}

File: decompiled/sources/com/hjq/http/EasyLog.kt
package com.hjq.http

class EasyLog private constructor() {
    companion object {
        @JvmStatic
        fun print() {
            print("----------------------------------------")
        }

        @JvmStatic
        fun print(str: String?) {
            if (EasyConfig.getInstance().isLogEnabled) {
                EasyConfig.getInstance().logStrategy.print(str)
            }
        }

        @JvmStatic
        fun json(str: String?) {
            if (EasyConfig.getInstance().isLogEnabled) {
                EasyConfig.getInstance().logStrategy.json(str)
            }
        }

        @JvmStatic
        fun print(str: String?, str2: String?) {
            if (EasyConfig.getInstance().isLogEnabled) {
                EasyConfig.getInstance().logStrategy.print(str, str2)
            }
        }

        @JvmStatic
        fun print(th: Throwable) {
            if (EasyConfig.getInstance().isLogEnabled) {
                EasyConfig.getInstance().logStrategy.print(th)
            }
        }

        @JvmStatic
        fun print(stackTraceElementArr: Array<StackTraceElement>) {
            if (EasyConfig.getInstance().isLogEnabled) {
                EasyConfig.getInstance().logStrategy.print(stackTraceElementArr)
            }
        }
    }
}
